### DISCUSSION TITLE
Kerberos authentication for Apache Kafka scaler

### DISCUSSION THREAD

**Discussion by sonialas** [kedacore/keda#2673] [Q&A / Need Help] [answered]
I would like to know if there is any way to configure an **authentication with Kerberos** for the **Apache Kafka scaler**, i.e. is there any way to specify the kerberos configuration files with the current configuration? Is this update planned?

Thank you!

**Comment by tomkerkhove** [comment, marked as answer]
All our current authentication is documented on https://keda.sh/docs/latest/scalers/apache-kafka/#authentication-parameters but I don't think we support this today.

We have https://github.com/kedacore/keda/issues/857 on the backlog, would that help?

**Reply by zroubalik** [reply]
If this is something that is supported by https://github.com/Shopify/sarama (which we use as a Kafka Go library), then the support on KEDA side should be relatively easy to implement.

**Reply by sonialas** [reply]
@tomkerkhove Thank you for your response!

Regarding the supported authentication parameters, within TLS, could you specify certificates that work as Mutual TLS? is the right way to implement MTLS?

Regarding the backlog issue for Kerberos, we will check your information @zroubalik  anf if we can help to implement it. 

Best regards

**Reply by zroubalik** [reply]
You can specify certs if you want to use TLS, it in examples section in the docs https://keda.sh/docs/2.6/scalers/apache-kafka/
